Start Docs directory (#1)
hashicorp/structure-design-system: README.md

# Structure Design System

Structure is a Design System for HashiCorp Web UI projects.

## Getting Started

#### Read the Documentation

- [Status](docs/status.md)
- [Components](docs/components)

#### Design

Our Design System is designed in Sketch and [stored in Abstract](https://share.goabstract.com/0b159f72-b8fb-4441-9ccc-65a1a4ec995d) for sharing and organizing. If you don’t have access to Abstract, ask in [#proj-product-ds](https://hashicorp.slack.com/messages/C7KTUHNUS/).

## Contributing

#### What should go in Structure?

> The most obvious candidate is if we are using the same design pattern (or similar variants) in multiple products. If you have a component you think we should be using or think that we could be doing something better, these are also good bets. Not sure? Ask in [#proj-product-ds](https://hashicorp.slack.com/messages/C7KTUHNUS/)

Reviewer: ```suggestion
The most obvious candidate are those design patterns (or similar variants) used in multiple products. If you have a component you think we should be using or think that we could be doing something better, these are also good bets. Not sure? Ask in [#proj-product-ds](https://hashicorp.slack.com/messages/C7KTUHNUS/)
```
Just simplified the first sentence -- read a little odd to me.
